app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -292,11 +293,15 @@ fun HikeDetails(
               value = "${elevationGainString}m")
           DetailRow(
               label = stringResource(R.string.hike_detail_screen_label_estimated_time),
-              value = "${hourString}:${minuteString}")
+              value = "${hourString}h${minuteString}")
```

Comment:
Question: I find it weird that when the hike is less than one hour 00h25m is displayed. Wouldn't it be better to display 25m instead? Do you know if it is possible to do so?

Reply:
Good point, thats not too difficult to implement. I've changed it to be 00min if the time is less than an hour